From ff579a4828e472689a9f3618d318c5b9b79bdfed Mon Sep 17 00:00:00 2001
From: MSe <mse1969@posteo.de>
Date: Fri, 8 Jun 2018 17:33:06 +0200
Subject: [PATCH 202/249] manifest: Track own external/libvorbis

Google has not published most recent 8.1.0_r30 tag for external/libvorbis
and instead declared the patch for CVE-2018-5146 "non-public".
Hence we need to track our own fork of it.

Change-Id: I3b9e27daf58ede43cff225faff43258d36c78a8e
---
 default.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default.xml b/default.xml
index 4e18e8c..f39fa71 100644
--- a/default.xml
+++ b/default.xml
@@ -216,7 +216,7 @@
   <project path="external/libusb" name="platform/external/libusb" groups="pdk" remote="aosp" />
   <project path="external/libusb-compat" name="platform/external/libusb-compat" groups="pdk" remote="aosp" />
   <project path="external/libvncserver" name="platform/external/libvncserver" groups="pdk" remote="aosp" />
-  <project path="external/libvorbis" name="platform/external/libvorbis" groups="pdk" remote="aosp" />
+  <project path="external/libvorbis" name="LineageOS/android_external_libvorbis" groups="pdk" />
   <project path="external/libvpx" name="platform/external/libvpx" groups="pdk" remote="aosp" />
   <project path="external/libvterm" name="platform/external/libvterm" groups="pdk" remote="aosp" />
   <project path="external/libxml2" name="platform/external/libxml2" groups="pdk,libxml2" remote="aosp" />
-- 
2.17.1

